src/components/molecules/ScheduleEventListNG/ScheduleEventListNG.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo } from "react";
+import isToday from "date-fns/isToday";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { isEventLive } from "utils/event";
+import { formatDateRelativeToNow } from "utils/time";
+
+import { ScheduleEventSubListNG } from "./ScheduleEventSubListNG";
+import { isEventLater, isEventSoon } from "./utils";
+
+import "./ScheduleEventListNG.scss";
+
+export interface ScheduleEventListNGProps {
+  daysEvents: PersonalizedVenueEvent[];
+  scheduleDate: Date;
+}
+
+export const ScheduleEventListNG: React.FC<ScheduleEventListNGProps> = ({
+  daysEvents,
+  scheduleDate,
+}) => {
+  const isTodayDate = isToday(scheduleDate);
+
+  const liveEvents = useMemo(() => daysEvents.filter(isEventLive), [
+    daysEvents,
+  ]);
+  const comingSoonEvents = useMemo(() => daysEvents.filter(isEventSoon), [
+    daysEvents,
+  ]);
+  const laterEvents = useMemo(() => daysEvents.filter(isEventLater), [
+    daysEvents,
+  ]);
```

Comment:
when these constants will be re-calculated? I believe we should use some setInterval to check if the event moved from status "coming soon" to "live" (for example).

Reply:
They will be re-calculated every time `daysEvents` are changed. Please clarify what you mean about `setInterval`?

I assume `isEventLive`, `isEventSoon` and `isEventLater` is used `setInterval` under the hood

Reply:
have you checked that event is moving from "coming soon" to "live" list automatically once it's started? 

My suggestion about setInterval was that we should check every minute/5 minutes what's the status of events to move them to the appropriate group, and if the event is finished, hide it. does that make sense?

Reply:
Got it. fixed in f3056b41c3cf3139daa66a9d68ca53e48c33f236